cmake/python_venv.cmake

```diff
@@ -0,0 +1,179 @@
+# (C) Copyright 2018- ECMWF.
+# This software is licensed under the terms of the Apache Licence Version 2.0
+# which can be obtained at http://www.apache.org/licenses/LICENSE-2.0.
+# In applying this licence, ECMWF does not waive the privileges and immunities
+# granted to it by virtue of its status as an intergovernmental organisation
+# nor does it submit to any jurisdiction.
+
+##############################################################################
+#.rst:
+#
+# find_python_venv
```

Comment:
Are we using this for the Loki setup, or is this just blindly copied? If not, please remove, as I think Loki does handle its own venvs.

Reply:
Removed